### DISCUSSION TITLE
How to optimize own consumption of Deye 12k LV inverter

### DISCUSSION THREAD

**Discussion by ap20132022** [kellerza/sunsynk#358] [Q&A]
I have a Deye 12k inverter. As I found out that this inverter has a "high" own power consumption (120W), I want to optimize it. 
If I see that my battey in the evening only <15%, I changed the configuration "Battery type" to "No battery". I couldn't see any improvement and the inverter shows and sounds alarm. 
Do you have any suggestion/ideas for me? 
Thanks a lot.

**Comment by ap20132022** [comment]
does anyone have an idea?

**Comment by kellerza** [comment]
Maybe if the soc target is high you will only pass through the grid, and not draw from the battery (make sure you don't charge to that soc using the grid)

**Reply by ap20132022** [reply]
> Maybe if the soc target is high you will only pass through the grid, and not draw from the battery (make sure you don't charge to that soc using the grid)

thank you for your reply. 
I meant the Standby consumption (~120W)
In german forum, there is a workaround with native Modbus over ESP32 in 2 steps:
- turn off the "power-on" - register 80
- turn off "beeps" - register 228

I couldn't find these register / value on this add on. Can you help me?
I'm using this add on with my home assistant and I love it. It's wasted to use one more software, in order t turn it off. 
Thank you very much and have a nice weekend.

**Reply by kellerza** [reply]
No idea what that does, but you can add those registers as 2 “custom sensors”/mysensors - see the docs

**Reply by ap20132022** [reply]
> No idea what that does, but you can add those registers as 2 “custom sensors”/mysensors - see the docs

Thank you very much for your suggest. I've ready theses instruction. 
I need to send write Command Register 80 & 228
I wonder how I can add custom WRITE command?

**Reply by tafx** [reply]
> > No idea what that does, but you can add those registers as 2 “custom sensors”/mysensors - see the docs
> 
> Thank you very much for your suggest. I've ready theses instruction. I need to send write Command Register 80 & 228 I wonder how I can add custom WRITE command?

Could you please post the link to original information about standby consumption workaround? I can not find it. Thanks.

**Reply by ap20132022** [reply]
it was on a german forum akkudoktor. Pls. google it.
